src/elements/content-explorer/stories/tests/ContentExplorer-visual.stories.js

```diff
@@ -1,25 +1,152 @@
 import { http, HttpResponse } from 'msw';
+import { expect, userEvent, waitFor, within, screen } from '@storybook/test';
 
-import { userEvent, waitFor, within } from '@storybook/test';
 import ContentExplorer from '../../ContentExplorer';
-import { DEFAULT_HOSTNAME_API } from '../../../../constants';
 import mockRootFolder from '../__mocks__/mockRootFolder';
-import { defaultVisualConfig, SLEEP_TIMEOUT } from '../../../../utils/storybook';
+import mockSubFolder from '../__mocks__/mockSubFolder';
+import { defaultVisualConfig } from '../../../../utils/storybook';
+
+import { DEFAULT_HOSTNAME_API } from '../../../../constants';
 
-export const basic = {};
+export const basic = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        expect(await canvas.findByText('An Ordered Folder')).toBeInTheDocument();
```

Comment:
use asynchronous function findByText() at first to make sure the initial page has rendered.